src/aere_debugger.erl

```diff
@@ -0,0 +1,119 @@
+-module(aere_debugger).
+
+-export([ add_breakpoint/3
+        , delete_breakpoint/2
+        , resume_eval/2
+        , lookup_variable/2
+        , source_location/1
+        , stacktrace/1
+        ]).
+
+
+-spec add_breakpoint(ReplState, FileName, Line) -> ReplState
+    when ReplState :: aere_repl_state:state(),
+         FileName  :: string(),
+         Line      :: integer().
+
+add_breakpoint(State, FileName, Line) ->
```

Comment:
One can add a breakpoint in a file that does not exist. This is super prone to typos, I think we should restrict only to files that are loaded.

Reply:
Done here b593859aaec13f07a04c21ce9459f8e022d3c894